libraries/apollo-gradle-plugin/testProjects/multiplatform/build.gradle.kts

```diff
@@ -26,18 +26,3 @@ apollo {
         packageNamesFromFilePaths()
     }
 }
-
-// See https://youtrack.jetbrains.com/issue/KT-56019
-val myAttribute = Attribute.of("com.apollographql.test", String::class.java)
```

Comment:
I think we have more instances of this in `tests`

Reply:
Actually removing the ones in tests still fails. But this one produces `Configuration with name 'releaseFrameworkIosFat' not found.` and removing this resolves that.

Reply:
Interesting, looks like [it's not yet fixed with 1.9](https://youtrack.jetbrains.com/issue/KT-55751/MPP-Gradle-Consumable-configurations-must-have-unique-attributes#focus=Comments-27-7362740.0-0). Something else to try in the -RC

Reply:
Added a comment there: https://youtrack.jetbrains.com/issue/KT-55751/MPP-Gradle-Consumable-configurations-must-have-unique-attributes#focus=Comments-27-7417705.0-0